parsec/core/fs/exceptions.py

```diff
@@ -42,134 +42,132 @@ def __str__(self):
         return self.message
 
 
-# Base errors
+class FSWorkspaceNotFoundError(FSError):
+    pass
+
+
+class FSWorkspaceTimestampedTooEarly(FSError):
+    pass
```

Comment:
Why are `FSWorkspaceNotFoundError` and `FSWorkspaceTimestampedTooEarly` not defined in terms of `FSOperationLocalError`, `FSOperationRemoteError` or `FSInternalError`?

Reply:
55cd0f4430fd205d08ebecd47653113dfdf0bcdf